bin/build.sh

```diff
@@ -2,6 +2,7 @@
 #TODO: svn checkout, svn commit (if needed) after svn sync
 QUIET=1
 SYNC_SVN=0
+PATH_TO_COMPOSER="/usr/bin/php5.6 /usr/local/bin/composer "
```

Comment:
I can't build it. I don't have the same path to php5.6 that you

Reply:
Yes. It's explain on the linear ticket and on the release documentation. You need to adapt it for now